README.md

```diff
@@ -2,23 +2,22 @@
 
 WSO2 API Manager is a full lifecycle API Management solution which has an API Gateway and a Microgateway. 
 
-This explains how WSO2 API Manager plans to integrate with Ping Intelligence and expose APIs protected with 
-Artificial Intelligence.
+This explains how WSO2 API Manager plans to integrate with PingIntelligence and expose APIs protected with 
+artificial intelligence (AI).
 
-## API Manager Extension with Ping Intelligence
+## WSO2 API Manager Extension with PingIntelligence
 
 ### What is PingIntelligence for APIs?
-PingIntelligence for APIs uses artificial intelligence (AI) to expose active APIs, identify and automatically block cyber attacks on APIs and provide detailed reporting on all API activity. Deployed on premises, in public clouds or in hybrid clouds, the solution monitors API traffic across the environment. It uses AI and machine learning models to detect anomalous API behavior without relying on specifically defined policies or prior knowledge of attack patterns in which, can stop new and constantly changing attacks. Using continuous learning capabilities, it becomes more accurate at identifying and blocking attacks over time. 
+PingIntelligence for APIs uses artificial intelligence (AI) to expose active APIs, identify and automatically block cyber attacks on APIs and provide detailed reporting on all API activity. You can deploy the PingIntelligence solution on premises, in public clouds, or in hybrid clouds to monitor API traffic across the environment. PingIntelligence uses AI and machine learning models to detect anomalous API behavior, without relying on specifically defined policies or prior knowledge of attack patterns, in order to stop new and constantly changing attacks. In addition, PingIntelligence uses its continuous learning capabilities to become more accurate at identifying and blocking attacks over time. 
 
 PingIntelligence for APIs can detect many types of cyberattacks, most of which are not visible to API teams today and can go undetected for very long times. 
 
-[Read more about cyber attacks which can be detected by Ping Intelligence.](https://github.com/wso2-extensions/apim-handler-pingai/blob/master/DEVELOPER_GUIDE.md#pingintelligence-protects-against-three-main-types-of-attacks-specifically)
+[Read more about cyber attacks that can be detected by Ping Intelligence.](https://github.com/wso2-extensions/apim-handler-pingai/blob/master/DEVELOPER_GUIDE.md#pingintelligence-protects-against-three-main-types-of-attacks-specifically)
 
 ### How does integration happen?
+The WSO2 API Manager extension for PingIntelligence uses a new custom handler, namely Ping AI Security Handler, when working with the WSO2 API Gateway data flow. After this handler receives a request from a client, a sideband call is sent to PingIdentity’s API Security Enforcer (ASE) with the client request metadata. The ASE responds after analyzing the metadata with an Artificial Intelligence Engine. 
```

Comment:
Should 'namely Ping AI Security Handler, when working with the WSO2 API Gateway data flow' be 'namely Ping AI Security Handler, which is working with the WSO2 API Gateway data flow'?

Reply:
It is not correct to add "which" here, so to avoid confusion I updated the description by adding the name of the handler within brackets.